### PR TITLE
[13.0][MIG] website_livechat

### PR DESCRIPTION
**Pull request by AMhadhbi**
Description of the issue/feature this PR addresses:
Nothing to do

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---Models in module 'website_livechat'---
+---Fields in module 'website_livechat'---
+website_livechat / mail.channel             / livechat_active (boolean)     : NEW
+website_livechat / mail.channel             / livechat_visitor_id (many2one): NEW relation: website.visitor
+website_livechat / website.visitor          / livechat_operator_id (many2one): NEW relation: res.partner, isfunction: function, stored
+website_livechat / website.visitor          / mail_channel_ids (one2many)   : NEW relation: mail.channel
+# Nothing to do
+---XML records in module 'website_livechat'---
+DEL im_livechat.channel: website_livechat.im_livechat_channel_data_website (noupdate)
```
Comment (pedrobaeza) [inline addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:9]: This is in fact renamed to `im_livechat.im_livechat_channel_data` (but in `im_livechat`), so it's interesting to mark it.

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---Models in module 'website_livechat'---
+---Fields in module 'website_livechat'---
+website_livechat / mail.channel             / livechat_active (boolean)     : NEW
+website_livechat / mail.channel             / livechat_visitor_id (many2one): NEW relation: website.visitor
+website_livechat / website.visitor          / livechat_operator_id (many2one): NEW relation: res.partner, isfunction: function, stored
```
Comment (pedrobaeza) [inline addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:5]: I think you should fill this in a similar manner as https://github.com/OCA/OpenUpgrade/pull/2321/files#diff-731049fb90a841c7cf2ae3ec345b644ab76cffd65af083c32f9ac5dbbbba08b4R15

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (AMhadhbi) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:9]: I marked as : # DONE in im_livechat : renamed to im_livechat.im_livechat_channel_data, right?

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 2 above)
Comment (AMhadhbi) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:5]: @pedrobaeza i added this function
Screenshot 2021-02-10 at 20 52 34
website.visitor is new model so why we need to update it? i think there is no changes needed! or should i insert into table website_visitor with livechat_operator_id and livechat_visitor_id
Thanks.

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (pedrobaeza) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:9]: I would say that `# NOTHING TO DO: Renamed to im_livechat.im_livechat_channel_data in im_livechat`

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 2 above)
Comment (pedrobaeza) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:5]: OK, I see, thanks for the insight. No need of doing anything indeed, as it's a new model, so no records in it, and guessing them is not suitable.

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (AMhadhbi) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:9]: Done.

File: addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt (hunk 2 above)
Comment (AMhadhbi) [reply addons/website_livechat/migrations/13.0.1.0/openupgrade_analysis_work.txt:5]: Thanks for your feedback.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**